helm-chart/eoapi/templates/pgstacboostrap/job.yaml

```
@@ -51,5 +51,13 @@ spec:
         - name: pgstac-setup-volume-{{ $.Release.Name }}
           configMap:
             name: pgstac-setup-config-{{ $.Release.Name }}
+      {{- with .Values.pgstacBootstrap.settings.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.pgstacBootstrap.settings.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
```

Can we check for the existence of `affinity` and `tolerations`, and possibly avoid adding them as empty objects to `values.yaml`?

```suggestion
      {{- if .Values.pgstacBootstrap.settings.affinity }}
      affinity:
        {{- toYaml .Values.pgstacBootstrap.settings.affinity | nindent 8 }}
      {{- end }}
      {{- if .Values.pgstacBootstrap.settings.tolerations }}
      tolerations:
        {{- toYaml .Values.pgstacBootstrap.settings.tolerations | nindent 8 }}
      {{- end }}
```